src/Corporation/Actions.ts

```diff
@@ -463,7 +463,14 @@ export function Research(researchingDivision: Division, researchName: CorpResear
   const researchTree = IndustryResearchTrees[researchingDivision.type];
   if (researchTree === undefined) throw new Error(`No research tree for industry '${researchingDivision.type}'`);
   const research = ResearchMap[researchName];
-
+  const child = researchTree.findNode(researchName);
```

Comment:
`child` is not a great variable name here, it's a research tree node associated with the "research" variable, so I would name it `researchNode`.